cola/core.py

```diff
@@ -143,8 +144,18 @@ def start_command(cmd, cwd=None, add_env=None,
         cwd = encode(cwd)
 
     if WIN32:
-        CREATE_NO_WINDOW = 0x08000000
-        extra['creationflags'] = CREATE_NO_WINDOW
+        # If git-cola is invoked on Windows using "start pythonw git-cola",
+        # a console window will briefly flash on the screen each time
+        # git-cola invokes git, which is very annoying.  The code below
+        # prevents this by ensuring that any window will be hidden.
+        startupinfo = subprocess.STARTUPINFO()
+        startupinfo.dwFlags = subprocess.STARTF_USESHOWWINDOW
+        startupinfo.wShowWindow = subprocess.SW_HIDE
+        extra['startupinfo'] = startupinfo
+
+        if WIN32 and not no_win32_startupinfo:
+            CREATE_NO_WINDOW = 0x08000000
+            extra['creationflags'] = CREATE_NO_WINDOW
```

Comment:
This is a great solution.  I'll merge the early part first and we can follow-up with the later parts since they have some nice usability improvements.